dist/jdataview.js

```diff
@@ -1,23 +1,23 @@
-(function (factory) {
-	var global = this;
-
-	if (NODE || typeof exports === 'object') {
-		module.exports = factory(global);
-	} else
-	if (BROWSER) {
-		if (typeof define === 'function' && define.amd) {
-			define([], function () {
-				return factory(global);
-			});
-		}
-		else {
-			global.jDataView = factory(global);
-		}
-	}
-}(function (global) {
-
-'use strict';
-
+(function (factory) {
```

Comment:
Fair enough - initially it was just used in some demos via rawgit, and I didn't want to break them. And now, it doesn't matter much given that project is basically "finished" and changes don't occur often enough.